src/app/forms/models/validators.enum.ts

```diff
@@ -29,5 +29,6 @@ export enum ValidatorNames {
   NotZNumber = 'notZNumber',
   MustEqualSibling = 'mustEqualSibling',
   HandlePsvPassengersChange = 'HandlePsvPassengersChange',
+  Enum = 'enum',
```

Comment:
can we get a more descriptive name for this? I assume it's validating it is an enum but

Reply:
Yeah it checks if the form control value is a member of an enum. Would IsMemberOfEnum be okay?

Reply:
Yeah I like that a lot more! Rest of PR looks good, just pulled it down and played with it too, looks good :)